drivers/gpu/drm/tests/Makefile

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: GPL-2.0-only
+obj-$(CONFIG_DRM_DAMAGE_HELPER_KUNIT_TEST) += test-drm_damage_helper.o
+obj-$(CONFIG_DRM_CMDLINE_PARSER_KUNIT_TEST) += test-drm_cmdline_parser.o
+obj-$(CONFIG_DRM_RECT_KUNIT_TEST) += test-drm_rect.o
+obj-$(CONFIG_DRM_FORMAT_KUNIT_TEST) += test-drm_format.o
+obj-$(CONFIG_DRM_PLANE_HELPER_KUNIT_TEST) += test-drm_plane_helper.o
+obj-$(CONFIG_DRM_DP_MST_HELPER_KUNIT_TEST) += test-drm_dp_mst_helper.o
+obj-$(CONFIG_DRM_FRAMEBUFFER_KUNIT_TEST) += test-drm_framebuffer.o
+obj-$(CONFIG_DRM_BUDDY_KUNIT_TEST) += test-drm_buddy.o
+obj-$(CONFIG_DRM_MM_KUNIT_TEST) += test-drm_mm.o
```

Comment:
Acho que faltou atualizar esses paths quando rolou o rename `test-*` para `*_test`

Reply:
My bad! I will fix (and also turn it into single entry as requested by Javier)

Reply:
Fixed!